core/domain/exp_domain.py

```diff
@@ -3671,6 +3671,24 @@ def get_all_html_content_strings(self):
 
         return html_list
 
+    def get_all_html_content_strings_with_state_name(self):
+        """Gets all html content strings used in this exploration with the
```

Comment:
I think maybe you can call this get_all_html_strings() and similar for state.interaction.get_all_html_strings(), since "content" has a specific meaning -- it relates to a specific part of a state?

Reply:
Hi @seanlip, I've addressed this one. I think it is ready to be run on the test server.